jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/projects/jakarta-sample/src/main/java/io/openliberty/sample/jakarta/jsonb/JsonbTransientDiagnostic.java

```diff
@@ -35,7 +37,7 @@ private int getId() {
         return id;
     }
     
-    @JsonbCreator
+    @JsonbNillable
```

Comment:
Just curious why this was modified?

Reply:
Ohh, the reason is that `@JsonbCreator` annotation is only used for constructors, so just changed it to follow the actual usage. :)

Reply:
Great, thank you for catching this.